app/Slack/Integration.php

```diff
@@ -6,8 +6,17 @@
  * @property int $id {primary}
  * @property string $name
  * @property string $hookUrl
+ * @property string|null $channel
```

Comment:
Osobně bych se nebál to udělat povinné s default hodnotou

Reply:
To je zajimava otazka, me se moc nelibi defaultni hodnoty drzet v databazi, stejne jako enumy. Za me by takhle logika (defaultni hodnota) mela byt drzena spis v kodu, je to jednodussi na spravu a take vidime verzovani.

Reply:
Chápu, pak tedy pokud budeme mít `getterChannel(): string`, který zaručí, že se vždy vrátí string i po zavolání `$integration->channel`, tak se to trochu zkomplikuje. Mohli bychom nechat v databázi jen string, ale on se může uložit i prázdný string. Leda ho kontrolovat v tom `getterChannel()` podobně, jako `NULL`. Co?

Reply:
Ted to mam napsany takhle

```
	protected function getterChannel(?string $channel): string
	{
		if ($channel) {
			return $channel;
		}

		return self::DEFAULT_CHANNEL;
	}
```

Ohledne prazdneho stringu uprimne nevim, prazdny string je proste spatna hodnota, stejne jako nazev kanalu bez mrizky.

Reply:
Kazdopadne neni to strictni kontrola, ale prazdny string v getteru co jsem poslal vyse, taky neprojde.